fleet-cmdr/src/handlers/cmd_orb_details.rs

```diff
@@ -0,0 +1,49 @@
+use async_trait::async_trait;
+use color_eyre::eyre::Result;
+use orb_relay_client::{QoS, RecvMessage};
+use orb_relay_messages::{
+    orb_commands::v1::{OrbCommandError, OrbDetailsRequest, OrbDetailsResponse},
+    prost::Message,
+};
+use tracing::info;
+
+use super::OrbCommandHandler;
+
+#[derive(Debug)]
+pub struct OrbDetailsCommandHandler {}
+
+impl OrbDetailsCommandHandler {
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+#[async_trait]
+impl OrbCommandHandler for OrbDetailsCommandHandler {
+    #[tracing::instrument]
+    async fn handle(&self, command: &RecvMessage) -> Result<(), OrbCommandError> {
+        info!("Handling orb details command");
+        let _request = OrbDetailsRequest::decode(command.payload.as_slice()).unwrap();
+        // TODO(paulquinn00): Consult with @oldgalileo and @sfikastheo to determine where to get this info from.
+        let response = OrbDetailsResponse {
```

Comment:
Many of these are unfortunately fail-able. I would _**heavily**_ suggest that we structure this such that there's an `OrbDetailsResponse` builder which returns a Result and that we notify the backend in the response which fields have failed based off of that Result.

We should be more aggressive about tracking issues with orb-names not existing, jabil-id's not existing, issues with the MCU version reporting, etc.

Reply:
@paulquinn00 add a todo in the code that links to a linear ticket and then I'll merge